lib/config/options/index.ts

```diff
@@ -859,7 +859,7 @@ const options: RenovateOptions[] = [
   {
     name: 'gitIgnoredAuthors',
     description:
-      'Additional Git authors which are ignored by Renovate. Must conform to [RFC5322](https://datatracker.ietf.org/doc/html/rfc5322).',
+      'Git authors which are ignored by Renovate. Must conform to [RFC5322](https://datatracker.ietf.org/doc/html/rfc5322).',
```

Comment:
Does improving the `description` here make it a `fix:` or `refactor:` PR?

Reply:
no, it'll normally only printed to docs and logs. but can be visible on onboarding PR.